tests/test_slicing.py

```diff
@@ -21,3 +26,75 @@ def test_layernorm_fusion_replaces_modules() -> None:
 
 def get_module_names(model) -> list[str]:
     return [name for name, _ in model.named_parameters()]
+
+
+def test_HF_model():
+    """Check that the HF model weights are equivalent to the sliced model weights"""
+    model_name = "microsoft/phi-2"
+    model_adapter, tokenizer = hf_utils.get_model_and_tokenizer(model_name)
+    sparsity = 0.1
+
+    layernorm_fusion.replace_layers(model_adapter)
+    layernorm_fusion.fuse_modules(model_adapter)
+
+    config = PhiConfig.from_pretrained(
```

Comment:
If we need a full Phi2 for the test, then we need the pytest markers like the ones in `test_experiments.py`.